### PR TITLE
WordContentHandler: Fix span removal in strange lists.

### PR DESCRIPTION
**Pull request by tpusztaiGentics**
When some paragraphs are formatted as ListParagraph in Word, but have no bullets, numbers, letters, etc., these paragraphs might not contain the span element that is normally used to detect the list type and which is removed after detection (this causes an entire paragraph to be removed in such a case). Only remove this span if it either has an "mso-list: Ignore" style or if it is nested within a  comment.

SUP-1286

Admin please build this request

### CODE REVIEW

**Comment by GenticsDev** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.office:8080//job/alohaeditor-upstream-pullrequests/302/
Test PASSed.

**Comment by GenticsDev** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.office:8080//job/alohaeditor-upstream-pullrequests/310/
Test PASSed.

File: src/plugins/common/contenthandler/lib/wordcontenthandler.js [hunk 1]
```diff
@@ -273,6 +273,25 @@ define([
...
 		},
+		
+		/**
+		 * Checks if the specified node is enclosed in a <!--[if !supportLists]--> comment.
+		 * @param node The Node, whose position should be checked.
+		 * @param stopAt An ancestor Node of node, where the search will be stopped (this node will not be searched any more).
+		 * @return true if the node is enclosed in a <!--[if !supportLists]--> comment within the stopAt node, otherwise false.
+		 */
+		checkElementIsEnclosedInListsComment: function (node, stopAt) {
+			while (node !== stopAt) {
```
Comment (npomaroli) [inline src/plugins/common/contenthandler/lib/wordcontenthandler.js]: also leave loop, if node is undefined

**Comment by GenticsDev** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.office:8080//job/alohaeditor-upstream-pullrequests/312/
Test PASSed.

**Pull request merged**